buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -518,6 +568,10 @@ public String getTransportPortURI() {
         return getTransportPortInternal();
     }
 
+    public File getServerLog() {
+        return confPathLogs.resolve(getName().replaceAll("-[0-9]+$", "") + "_server.json").toFile();
```

Comment:
Can we just use `safeName()` here?

Reply:
We should, but still need the replaceAll as the file has the cluster name in it not the node name, so we remove the numbering since we don't store the cluster name for nodes. We should probably revisit this at a lather time